floris/optimization/layout_optimization/layout_optimization_gridded.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+import numpy as np
+
+from .layout_optimization_base import LayoutOptimization
+from .layout_optimization_random_search import test_point_in_bounds
+
+
+class LayoutOptimizationGridded(LayoutOptimization):
+    def __init__(
+        self,
+        fmodel,
+        boundaries,
+        min_dist: float | None = None,
+        min_dist_D: float | None = -1,
+        rotation_step: float = 5.0,
+        rotation_range: tuple[float, float] = (0.0, 360.0),
+        translation_step: float | None = None,
+        translation_step_D: float | None = -1,
+        translation_range: tuple[float, float] | None = None,
+        hexagonal_packing: bool = False,
+        enable_geometric_yaw: bool = False,
+        use_value: bool=False,
+
+    ):
```

Comment:
Need to add docstring

Reply:
This could also underline this optimization differs in objective from some of the others